Lombiq.NodeJs.Extensions/scripts/get-path.js

```diff
@@ -9,7 +9,7 @@ const process = require('process');
 const getCwd = require('./get-cwd');
 const getConfig = require('./get-config');
 
-const verbose = false;
+const verbose = true;
```

Comment:
This was just for debugging if I know it right. So you shouldn't need this (including this PR).